ui/app/components/app/connected-sites-list/connected-sites-list.component.js

```diff
@@ -89,7 +89,7 @@ export default class ConnectedSitesList extends Component {
                       }
                       {domainIsExpanded
                         ? <div className="connected-sites-list__domain-origin">
-                          { domain.extensionId ? t('extensionId', [domain.extensionId]) : domain.key }
+                          { domain.extensionId ? t('extensionId', [domain.extensionId]) : domain.name ? domain.key : '' }
```

Comment:
Nit: it would be better to split this into multiple lines, or even better move this to a variable so we can avoid using a nested ternary.